Sources/FluentKit/Query/Builder/QueryBuilder+Aggregate.swift

```diff
@@ -25,48 +25,56 @@ extension QueryBuilder {
         self.aggregate(.count, key, as: Int.self)
     }
 
+    // TODO: `Field.Value` is not always the correct result type for `SUM()`, try `.aggregate(.sum, key, as: ...)` for now
```

Comment:
Since these are public APIs, do you want to make these comments public DocC markup?

Reply:
I thought about it, but if you want the honest fact of the matter, if I added it for these, I'd end up making myself do the _entire_ set of documentation for each method (params, description, etc.), including all the other methods that aren't affected by the problem. That's probably a good hour worth of work to do really properly (especially the way I write doc comments, just look around at some of other bits of Fluent I've documented 🤨), and I don't have it to spare right this moment 😰. Best I can do is say that if someone happens to add it to this branch for me, I will both gratefully and delightedly accept it 😅.